html/del/demos/code/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Пример с кодом — <del> — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
```

Comment:
```suggestion
  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap">
```

Подключает моноширинный шрифт